shared/css/design-system-pegasus.scss

```diff
@@ -9,13 +9,19 @@ $border-radius: 4px;
   outline-offset: 2px;
 }
 
+@mixin hr-style {
```

Comment:
Can you give context about how this works? I don't see it doing anything

Reply:
Oops! It does nothing, I just forgot to delete it 🙃